src/js/profile/mobile/widget/GridView.js

```diff
@@ -87,6 +88,7 @@
 				MATRIX_REGEXP = /matrix\((.*), (.*), (.*), (.*), (.*), (.*)\)/,
 				DATA_ROLE = "data-role",
 				RESIZE_TIMEOUT = 300,
+				BORDER_SIZE = 16,
```

Comment:
is BORDER_SIZE fixed?

Reply:
Yes. The border size is fixed in widget. I moved this value from line 778/818